data/src/main/java/com/microsoft/azure/kusto/data/WellKnownDataSet.java

```diff
@@ -0,0 +1,8 @@
+package com.microsoft.azure.kusto.data;
+
+public enum WellKnownDataSet {
+    PrimaryResult,
+    QueryCompletionInformation,
+    TableOfContents,
+    QueryProperties
```

Comment:
Is this query properties or query statistics?

Reply:
The response holds a table with TableKind:"QueryProperties"

Reply:
for example
{
		"FrameType": "DataTable",
		"TableId": 0,
		"TableKind": "QueryProperties",
		"TableName": "@ExtendedProperties",
		"Columns": [
			{
				"ColumnName": "TableId",
				"ColumnType": "int"
			},
			{
				"ColumnName": "Key",
				"ColumnType": "string"
			},
			{
				"ColumnName": "Value",
				"ColumnType": "dynamic"
			}
		],
		"Rows": [
			[
				1,
				"Visualization",
				"{\"Visualization\":null,\"Title\":null,\"XColumn\":null,\"Series\":null,\"YColumns\":null,\"AnomalyColumns\":null,\"XTitle\":null,\"YTitle\":null,\"XAxis\":null,\"YAxis\":null,\"Legend\":null,\"YSplit\":null,\"Accumulate\":false,\"IsQuerySorted\":false,\"Kind\":null,\"Ymin\":\"NaN\",\"Ymax\":\"NaN\"}"
			]
		]
	}